client/src/shared/Footer/Footer.tsx

```
@@ -1,56 +1,61 @@
-              mettromoni.com
+              key={link.label}
+              href={link.href}
+              className="p-2 sm:p-3 rounded-full hover:opacity-80 transition-all duration-200 
+                         flex items-center justify-center bg-white/10"
+              aria-label={link.label}>
+              <link.icon className="w-4 h-4 sm:w-5 sm:h-5 md:w-6 md:h-6" />
             </a>
-          </div>
+          ))}
+        </div>
+        <hr></hr>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)